### ISSUE TITLE
Add support for only running Download or Upload Test

### ISSUE THREAD

**Issue by gaby** [code-inflation/cfspeedtest#132]
When the host has a very slow connection, the user may not care at all about their upload or download speed.

If I want to do a quick test, I may only care about Download speed, similar to what fast.com does.

Example usage: `--download`, `--upload`, default to doing both

Related to #131

**Comment by code-inflation** [comment]
Good idea!

**Comment by code-inflation** [comment]
Implemented in `v1.3.0` 
```
      --download-only
          Test download speed only
      --upload-only
          Test upload speed only
```

**Comment by gaby** [comment]
@code-inflation Thanks 💪
